.ci/testruns/default.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: testmachinery.sapcloud.io/v1beta1
+kind: Testrun
+metadata:
+  generateName: tm-extension-aws-
+  namespace: default
+spec:
+
+  ttlSecondsAfterFinished: 172800 # 2 days
+
+  config:
+  - name: ACCESS_KEY_ID
+    type: env
+    valueFrom:
+      secretKeyRef:
+        name: shoot-operator-aws
+        key: accessKeyID
+  - name: SECRET_ACCESS_KEY
+    type: env
+    valueFrom:
+      secretKeyRef:
+        name: shoot-operator-aws
+        key: secretAccessKey
+
+  testflow:
+  - name: infrastructure-test
+    definition:
+      name: infrastructure-test
+      config:
+      - name: REGION
```

Comment:
I guess, the `$TM_KUBECONFIG_PATH` env var is always inject by the testmachinery, right?

Reply:
Yes

Reply:
yes exactly. Thats a default env variable

Reply:
As soon as this PR is merged.
You can run the test on PR by commenting `/test-single`